src/test/java/uk/gov/hmcts/reform/civil/sampledata/CaseDataBuilder.java

```diff
@@ -1292,6 +1292,20 @@ public CaseDataBuilder atStatePaymentSuccessful() {
         return this;
     }
 
+    public CaseDataBuilder atStatePaymentSuccessfulWithCopyOrganisationOnly() {
+        atStateClaimSubmitted();
+        claimIssuedPaymentDetails = PaymentDetails.builder()
+            .status(SUCCESS)
+            .reference("RC-1604-0739-2145-4711")
+            .build();
+        paymentReference = "12345";
```

Comment:
Why do we have these static values?

Reply:
Our test CaseDataBuilder is full of static values since its data we create for our tests. In my latest push i've changed I am now calling atStatePaymentSuccessful(); to populate the above static values.